vobject/icalendar.py

```diff
@@ -1920,6 +1920,8 @@ def error(msg):
                 sec = 0
             elif char == "eof":
                 state = "end"
+                if not current:
+                    second = '0'
```

Comment:
I don't understand how this change relates to the commit message. Esp. why are only th seconds set to zero?

Please either change the commit message or add a comment. Thx